_posts/2023-03-12-gs_00008_programming_II.md

```diff
@@ -0,0 +1,188 @@
+implement such a question you can use an **if-else-statement**:
+
+    if pancake is delicious:
+        recommend the recipe
+    else:
+        delete the recipe
+
+The if statement checks whether a condition is true. In our example, it is
+checked whether the pancake is delicious. If that condition is true, the
+instructions indented below the if statement will be executed, in our case
+recommending the recipe. If the pancakes don't taste good, the instructions
+below the else statement will be executed, here, deleting the recipe.
+
+You might say that this is too easy and this can't be all. Well... Yes and No!
+Of course, there is much more to learn. Otherwise, software engineers wouldn't
+be paid that well. But with the principles presented above, you can already
+construct very complex programs.
+
+## Programming beginners FAQ
+
+Before we move on, let us answer some common questions asked by programming
+beginners.
+
+<div class='note-exercise' markdown="1">
+Which programming language should I start with?
+</div>
+
+There is no general answer to this as it highly depends on what you want to do.
+In this blog series, we will teach Python and C. Why did we choose these? Python
+is very flexible and you can do almost everything with it. You can process data,
+build graphical applications, work with artificial intelligence, and much more.
+C is, for example, an ideal choice in combination with microcontrollers that are
+often used by programming hobbyists for their home automation projects.
+Moreover, both languages are pretty well established in the community for ages.
+So, there exists an almost infinite number of public projects and you can get a
+lot of help in forums. But in general, you should decide on your project first
+and then do some research to see which language suits the best.
+
+<div class='note-exercise' markdown="1">
+Is it hard to learn a programming language?
+</div>
+
+No! You'll see that it does not take long to get your first program running.
+From there on it is just experience. The more you code the better you get. We'll
+guide you to make it as easy as possible for you.
+
+<div class='note-exercise' markdown="1">
+What is the best way to learn a programming language?
+</div>
+
+Get your fingers on a keyboard! Often we hear people say "First, I buy a book
+and learn everything." Forget about it! Programming languages have almost
+unlimited possibilities and even experienced programmers don't know all of them.
+Start with some small basic tutorials like this blog series. Then, look for your
+own project and a search engine of your choice will help you to succeed.
+
+<div class='note-exercise' markdown="1">
+Math really sucks! Can I still learn to program?
+</div>
+
+Why do people always ask this??? Of course, when you write a program that shall
+do your maths jobs, then you should know how maths works. Otherwise, it would be
+hard to write the program. Besides that, mathematical knowledge is not required.
```

Comment:
I'm not sure about the statement that "math is not required" for programming. Maybe add something like:

> "But if you know basic maths or even are good at it then it can help you in programming, as both share common principles such as logical thinking and problem solving. But again, you don't need to study a whole maths book just to get started with programming!"

What do you think? You could also write it shorter, but I believe without it this addition, it really sounds like maths is absolutely not required, which I think is not fully true -- there is a reason why computer science has maths modules in its curriculum :)

Reply:
I think I'll just leave that question out